src/state/reducers/index.js

```diff
@@ -1,22 +1,5 @@
-import { combineReducers } from 'redux';
```

Comment:
One question I have about this, could someone not just import these reducers directly if they are provided from the package exports without this refactor?

Reply:
not clear on exactly what you mean.  they are not exported with names currently so they are not available to the consuming implementation.  

the benefit of named imports is [here](https://github.com/ProjectMirador/mirador/blob/ed33b9c61b96f38636ff44f3dc3e5e2236a9cd3f/src/state/reducers/rootReducer.js#L2-L4).  clean abbreviated syntax for imports.